packages/issue-receiver/__tests__/main.test.ts

```diff
@@ -0,0 +1,3 @@
+test('dummy', async () => {
```

Comment:
`jest` yelled at me when I didn't include any tests.

Reply:
That's correct. Alternatively you may:  `-passWithNoTests`